src/cli-util.js

```diff
@@ -33,6 +34,21 @@ function getOptions(argv) {
   };
 }
 
+function getRequirePragmaOption(argv) {
+  const value = argv["require-pragma"];
+
+  if (value === undefined) {
+    return value;
```

Comment:
Can you be explicit and `return undefined;`, otherwise it's not instantly clear that `value` is `undefined`